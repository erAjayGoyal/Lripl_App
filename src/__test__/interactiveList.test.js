// Link.react.test.js
import React from 'react';
import ineractiveList from '../components/InteractiveList';
import renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {
    let buttonConfig = ['edit', 'delete']
    let data = ["Category Name 1", "Category Name 2", "Category Name 3","Category Name 4", "Category Name 5"];
    
    
    
    const component = renderer.create(
    <ineractiveList
    buttonConfig = {buttonConfig}
    data = {data}
   // history = {this.props.history}
    pageName = "category"
    />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   tree.props.onMouseEnter();
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   tree.props.onMouseLeave();
//   // re-rendering
//   tree = component.toJSON();
//  expect(tree).toMatchSnapshot();
});