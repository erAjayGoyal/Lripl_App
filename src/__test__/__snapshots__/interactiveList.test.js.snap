// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<ineractiveList
  buttonConfig={
    Array [
      "edit",
      "delete",
    ]
  }
  data={
    Array [
      "Category Name 1",
      "Category Name 2",
      "Category Name 3",
      "Category Name 4",
      "Category Name 5",
    ]
  }
  pageName="category"
/>
`;
